apps/framework-cli/src/cli/routines/flow.rs

```diff
@@ -76,12 +76,75 @@ pub fn create_flow_file(
             )
         })?;
 
+    show_message!(
+        MessageType::Success,
+        Message {
+            action: "Created".to_string(),
+            details: "flow".to_string(),
+        }
+    );
+
+    let mut missing_datamodels = Vec::new();
+    if verify_datamodel(project, source.clone()).is_err() {
+        missing_datamodels.push(source);
+    }
+    if verify_datamodel(project, destination.clone()).is_err() {
+        missing_datamodels.push(destination);
+    }
+    if !missing_datamodels.is_empty() {
+        {
+            show_message!(
+                MessageType::Info,
+                Message {
+                    action: "".to_string(),
+                    details: "\n".to_string(),
+                }
+            );
+        }
+
+        let missing_datamodels_str = missing_datamodels.join(", ");
+        show_message!(
+            MessageType::Highlight,
+            Message {
+                action: "Next steps".to_string(),
+                details: format!(
+                    "You may be missing the following datamodels. Add these to {}: {}",
+                    project.schemas_dir().display(),
+                    missing_datamodels_str
+                )
+            }
+        );
+    }
+
     Ok(RoutineSuccess::success(Message::new(
-        "Created".to_string(),
-        format!("flow {}", flow_file_path.display()),
+        "".to_string(),
+        "".to_string(),
     )))
 }
 
+pub fn verify_datamodel(project: &Project, datamodel: String) -> anyhow::Result<()> {
+    let child = std::process::Command::new("grep")
+        .arg("-r")
+        .arg("--include=*.ts")
+        .arg("-E")
+        .arg(format!("export\\s+interface\\s+\\b{}\\b", &datamodel))
+        .arg(project.schemas_dir())
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .spawn()?;
```

Comment:
One option is to crawl the schema, get framework objects, and check against that. It seems heavy for just an init command.

Another option is to check the .moose folder, but that requires dev to have been run.

Grep seems lightweight enough and works on Mac/Linux. LMK what y'all think.

Reply:
That's a creative solution!

Quick question - should we fail if the datamodel doesn't exist?

Reply:
Their order of operation could vary. They could create their data models first, then the flow, in which case we wouldn't print next steps. They could create the flow file first, then their data models, in which case we're just reminding them to add their data models by printing these next steps. I think that's a valid workflow. Feels like the reminder is gentler & they won't have to come back & re-run the flow init command.

Reply:
🤔  that feels fair for the output but does it work for the input? ie can you write a flow without having defined the data you will take as input?

Reply:
Your other PR should cover this